concent_api/core/tests/test_unit_validation.py

```diff
@@ -168,3 +175,79 @@ def test_that_validation_should_raise_exception_when_checksum_is_invalid(self):
             with self.assertRaises(HashingAlgorithmError) as context:
                 validate_secure_hash_algorithm(invalid_value)
             self.assertEqual(context.exception.error_code, error_code)
+
+
+@override_settings(CONCENT_PUBLIC_KEY=CONCENT_PUBLIC_KEY)
+class TestAreEthAddressesAndSingularitiesEqual(ConcentIntegrationTestCase):
```

Comment:
"singularity" is a property of a value (I think "uniqueness" would be a better word here though). You're not comparing singularities. You're checking if the values are singular (unique). The name of the class does not make sense.

Reply:
done